docker/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.9-alpine
+FROM python:3.9.10-alpine
```

Comment:
Doesn't `python:3.9-alpine` always take the latest 3.9? Are we trying to avoid a particular version here or was it not taking 3.9.10?